sql/moz-fx-data-shared-prod/mozilla_vpn_derived/subscription_events_live/view.sql

```diff
@@ -1,89 +1,124 @@
 CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.mozilla_vpn_derived.subscription_events_live`
 AS
-WITH json_arrays AS (
+WITH all_subscriptions AS (
   SELECT
     *,
     TO_JSON_STRING(promotion_codes) AS json_promotion_codes
   FROM
-    `moz-fx-data-shared-prod`.mozilla_vpn_derived.all_subscriptions_v1
+    mozdata.mozilla_vpn.all_subscriptions
+),
+max_active_date AS (
+  SELECT AS VALUE
+    MAX(active_date)
+  FROM
+    mozdata.mozilla_vpn.active_subscription_ids
+),
+new_events AS (
+  SELECT
+    active_date AS event_date,
+    subscription_id,
+    "New" AS event_type,
+  FROM
+    mozdata.mozilla_vpn.active_subscription_ids
+  WHERE
+    TRUE -- zetasql requires QUALIFY to be used in conjunction with WHERE, GROUP BY, or HAVING
```

Comment:
Looks like they do that because `QUALIFY` isn't a reserved keyword in BigQuery?  That's annoying.

Reply:
to be clear, BigQuery allows it. Only zetasql is complaining, but all our queries have to parse in zetasql for dependency checking.